packages/ra-core/src/actions/accumulateActions.ts

```diff
@@ -2,7 +2,19 @@ import { crudGetMany, crudGetMatching } from './dataActions';
 
 export const CRUD_GET_MANY_ACCUMULATE = 'RA/CRUD_GET_MANY_ACCUMULATE';
 
-export const crudGetManyAccumulate = (resource, ids) => ({
+export const crudGetManyAccumulate = (
+    resource: string,
+    ids: []
+): {
+    type: string;
+    payload: {
+        resource: string;
+        ids: [];
+    };
+    meta: {
+        accumulate: any;
```

Comment:
For the metas, we need to be a bit smarter. As each meta has a special signification (and triggers a particular side effect), we probably need a subtype for `accumulate`, to be reused everywhere the `accumulate` meta is present.